### ISSUE TITLE
zotonic-configfile: 'find_config: command not found' on SmartOS

### ISSUE THREAD

**Issue by AlainODea** [zotonic/zotonic#891]
zotonic-configfile does not appear to be working on SmartOS.

```bash
$ bin/zotonic configfile
/home/zotonic/zotonic/src/scripts/zotonic-configfile: line 21: find_config: command not found
```

**Comment by mworrell** [comment]
Deal :+1: 

Ping me when you are back from your holidays :)

**Comment by AlainODea** [comment]
@mworrell I tried **master** on SmartOS and I had no issues.  This appears to be resolved.

**Comment by mworrell** [comment]
Great! :)

**Comment by AlainODea** [comment]
This is still an issue but pull#946 fixes it in a portable way.

**Comment by mworrell** [comment]
Thanks, merged #946

**Comment by mworrell** [comment]
:+1:
